src/status_im2/contexts/syncing/events.cljs

```diff
@@ -105,7 +105,7 @@
                               (rf/dispatch [:syncing/update-role constants/local-pairing-role-sender])
                               (rf/dispatch [:hide-bottom-sheet])))]
     (when-not (and error (string/blank? error))
-      (let [key-uid    (get-in db [:profile/login :key-uid])
+      (let [key-uid    (get-in db [:profile/profile :key-uid])
```

Comment:
not sure why we need both but :profile/login gets wiped and so this is persistent which is needed for bioetrics & password approach